apps/src/code-studio/pd/form_components_func/FormController.jsx

```diff
@@ -114,8 +114,6 @@ const FormController = props => {
   const [showDataWasLoadedMessage, setShowDataWasLoadedMessage] = useState(
     applicationId
   );
-  const applicationStatusOnSave = 'incomplete';
-  const applicationStatusOnSubmit = 'unreviewed';
```

Comment:
These changed because now there are two potential statuses on submit –– instead, let the backend figure out which status it should be